lib/algo/AdaptiveDMD.C

```diff
@@ -0,0 +1,124 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
```

Comment:
Should the year be updated to 2022 in another PR, to keep this one simple?

Reply:
I think that would be better.